src/resmom/linux/mom_mach.c

```diff
@@ -8001,6 +8008,63 @@ pidcache_create(void)
 	return (pidcache_arena);
```

Comment:
not related to the current fix, but we do use pidcache_create(), shall we check for the return value of calloc() and set/not set pidcache_starttime accordingly...